proposal.md

```diff
@@ -593,12 +595,12 @@ dependencies.
 
 ## Links {#links}
 
+Documentation and guides from other R repositories:
+
+-   [R-universe Documentation](https://docs.r-universe.dev) - R-universe is 
+useful for sharing development versions of CRAN packages and as a CI/CD 
+platform.
 -   [Bioconductor Packages: Development, Maintenance, and Peer
     Review](https://contributions.bioconductor.org/)
 -   [rOpenSci Packages: Development, Maintenance, and Peer
     Review](https://devguide.ropensci.org/)
-    
```

Comment:
The [r-multiverse](https://r-multiverse.org/) might be also worth mentioning, as a way to have just one submission to a repository and no further (human) interaction. It doesn't have much guidance on development but it has detailed guidance about [policies and usage](https://r-multiverse.org/aup.html).